### DISCUSSION TITLE
This is great!

### DISCUSSION THREAD

**Discussion by Janne71** [clementprevot/aduro#22] [Q&A] [answered]
Hi
This is great, that you working on an integration for aduro burners.
But when i search for the repository in HACS i cant seem to find it?

**Comment by Janne71** [comment]
Found it and installed it. This is very nice. 
My burner is a Aduro h3 lux with an room temperature sensor.
Theres 2 sensors identifyed by the integration, but one is diconnected and the other got a message i dont understand. "quia et suscipit suscipit recusandae consequuntur expedita et cum reprehenderit molestiae ut ut quas totam nostrum rerum est autem sunt rem eveniet architecto"
Will be grateful if you could help me. 
Thanks
/Janne71

**Comment by clementprevot** [comment, marked as answer]
Hi @Janne71 

The Aduro integration is not ready yet. For the moment the only things that works is the discovery and the "connection" to the burner. But nothing more than that.
It's still a work in progress.

As for the message you see, it's just a placeholder text name "Lorem Ipsum".

You should definitely uninstall this integration and wait for me to finish it (I hope soon).
